product-service/src/getProductById.ts

```diff
@@ -0,0 +1,26 @@
+import db from "./db";
+
+export const getProductById = async (event, _context) => {
```

Comment:
the same

Reply:
Added return types for the functions (`APIGatewayProxyHandler` from `aws-lambda` library)